packages/amplify-category-auth/src/index.js

```diff
@@ -394,6 +395,36 @@ const executeAmplifyHeadlessCommand = async (context, headlessPayload) => {
         .then(getUpdateAuthRequestAdaptor(context.amplify.getProjectConfig().frontend, context.updatingAuth.requiredAttributes))
         .then(getUpdateAuthHandler(context));
       return;
+    case 'import':
+      if (projectHasAuth(context)) {
+        return;
+      }
+      await validateImportAuthRequest(headlessPayload);
+
+      const srvcMetaData = supportedServices.Cognito;
```

Comment:
naming: `serviceMetadata`, don't abbreviate, also metadata lowercase 'd', but this const is not really needed as `provider` below could be deconstructed from `supportedService.Cognito`

Reply:
done